interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -335,17 +339,27 @@ namespace cling {
       "\n"
       "   " << metaString << "U <filename>\t\t- Unloads the given file\n"
       "\n"
-      "   " << metaString << "I [path]\t\t\t- Shows the include path. If a path is given -"
+      "   " << metaString << "I [path]\t\t\t- Shows all include paths. If a path is given -"
+                             "\n\t\t\t\t  adds the path to the include paths\n"
+      "   " << metaString << "include [path]\t\t\t- Shows all include paths. If a path is given -"
```

Comment:
I would just mention `.I`, which is the most popular form, but up to @couet and @Axel-Naumann if they also want a mention to `.include`.

Reply:
In principle, they are both mentioned in https://root.cern/manual/first_steps_with_root/#special-interpreter-commands so I thought for the sake of completeness it might stay here as well IMO.

Reply:
Following the `(x|X)` above, what about `(I|include)`?

Reply:
good idea. applied!